src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
 // write code here
+
+const table = document.querySelector('.field > tbody');
+const buttons = document.querySelector('.container');
+
+function modifyColumns(rezising) {
+  switch (rezising) {
+    case 'append-row':
+      table.appendChild(table.querySelector('tr').cloneNode(true));
```

Comment:
Ensure that the table has at least one row before attempting to clone it. If the table is empty, `table.querySelector('tr')` will return `null`, leading to an error.